learn.md

```diff
@@ -17,28 +17,27 @@ permalink: /learn/
 <div class="col-sm-12 col-md-1 cLearnSteps">1</div>
 <div class="col-sm-12 col-md-11 cBoxContainer">
       <div class="cBallerina-Box">
-     <p>Download a Ballerina installer and make sure it’s working on your machine. </p>
-     <a class="cBallerina-io-Home-main-download-button" href="/downloads">Download Ballerina<p>Distributions available for <br>Linux, OS X, and Windows</p></a>
+     <p>Click below to download a Ballerina installer and use the downloaded installer to install Ballerina on your machine. </p>
```

Comment:
Do we need to say "Click below.." as it is clearly visible as a download button?

Reply:
Fixed.